### PR TITLE
Improve SQLite default settings

### PR DESCRIPTION
**Pull request by anze3db**
Hey! I really like your idea of simplifying the creation of new projects and I was wandering if you'd also like to add a better out of the box experience for SQLite.

The changes in the PR should make SQLite scale better in high concurrency environments ([some benchmarks](https://blog.pecar.me/django-sqlite-benchmark)) and remove the most common [SQLite gotchas](https://blog.pecar.me/sqlite-prod).

I also want to propose the same change to Django's startproject template, but I figured it would be a good idea to try this out on your project first. Let me know what you think!

### CODE REVIEW

**Comment by epicserve** [comment]
I'm open to the idea, but I would need this added to the 4.1 template as well and then the code that auto updates the templates would need to be updated to add that change, or maybe a better approach would be to modify the code that creates the new project to make those changes to the new project files.

**Comment by anze3db** [comment]
The `init_command` and `transaction_mode` options were only added in Django 5.1, so to get the same functionality in older Django versions we'd need to create [our own DatabaseWrapper class](https://blog.pecar.me/sqlite-django-config#3-fine-tune-your-sqlite-settings). I'm happy to generate it as well if you think it's worth the extra hassle. Do you think there are people using dj-beat-drop to create new projects on older versions?

In any case, I'll move the change from the template to a python function called by `create_new_project`.

**Comment by epicserve** [comment]
Yeah, I don't think we need bother with adding it for older versions of Django. So you could add some logic to add if the version is greater than 5.1.

File: tests/test_new_command.py [hunk 1]
```diff
@@ -47,6 +67,7 @@
         'SECRET_KEY="{{ secret_key }}"',
         "ALLOWED_HOSTS=",
         "DATABASE_URL=sqlite:///{{ project_dir }}/db.sqlite3",
+        "{% 5.1 %}" + SQLITE_OPTIONS_ENV,
```
Comment (anze3db) [inline tests/test_new_command.py:70]: `{% 5.1 %}` feels like a hack, so please let me know if you have a better idea for running asserts based on the Django version.

**Comment by anze3db** [comment]
@epicserve I've added the logic for both the env and non-env versions. Let me know if this works for you!

**Comment by epicserve** [comment]
@anze3db,

I made a couple commits and merged. I switched to using a tuple instead of prefixing with the `"{% 5.1 %}"` string. It seemed a little better that way to me.

I also switched to using a constant for the options, so you only have to change it in one place.

Also when the options are in a URL, I made sure the options where URL encoded.

**Pull request merged**